### DISCUSSION TITLE
Does couchdb run on Ubuntu 18.04?

### DISCUSSION THREAD

**Discussion by rkrajnik** [apache/couchdb#2954] [General] [answered]
It seems installing on Ubuntu 18.04 fails with "missing" dependencies such as curl and libicu57 (Ubuntu 18.04 installs libicu60.)  After installing curl (again) then I get a dependency error on  looking for libcurl3 and libicu57... thanks in advance.

**Comment by wohali** [comment, marked as answer]
Yes, absolutely it does. You may have mis-followed the installation instructions.

Follow these instructions for 18.04 ( taken from here: https://docs.couchdb.org/en/latest/install/unix.html )

```bash
$ sudo apt-get install -y gnupg ca-certificates
$ echo "deb https://apache.bintray.com/couchdb-deb bionic main" \
    | sudo tee /etc/apt/sources.list.d/couchdb.list
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys \
  8756C4F765C9AC3CB6B85D62379CE192D401AB61
$ sudo apt update
$ sudo apt install -y couchdb
```

**Comment by rkrajnik** [comment]
Thank you - I’ll give that a try.    
  

  
  
  

  
  
>
